app/models/manageiq/providers/kubernetes/container_manager.rb

```diff
@@ -41,6 +41,10 @@ def self.description
     @description ||= "Kubernetes".freeze
   end
 
+  def self.fonticon
+    @fonticon ||= "pficon-kubernetes".freeze
```

Comment:
To me, this is an ideal use case for https://github.com/drapergem/draper (no relation :smile:)